package.json

```diff
@@ -183,6 +183,7 @@
     "webpack-assets-manifest": "^5.1.0",
     "webpack-cli": "^4.9.2",
     "webpack-inject-plugin": "^1.5.3",
+    "webpack-subresource-integrity": "^5.2.0-rc.1",
```

Comment:
@bc-peng maybe we want to get a stable version here as compared to a release candidate.